item/item_test.go

```diff
@@ -26,7 +26,7 @@ func TestRelativeTime(t *testing.T) {
 			item: Item{
 				UnixPostTime: 1739145600,
 			},
-			want: "13d",
```

Comment:
This is not good if you have to change that everyday.
In a future PR, please address this.